betovarg/lucysanchez
# LS-5: homepage gallery

## Steps to test

- go the the repository branch `feature/LS-5-homepage-gallery`.
- clean caches of Grav on admin Grav --> bin/grav clear-cache.
- go to localhost:8080/.
- review if the Gallery appears and have the ligthbox function.
- also check if navigation on lightbox is enabled.
- check if on admin page you can edit the gallery.
